Kafka/KafkaJCAAPI/src/main/java/fish/payara/cloud/connectors/kafka/inbound/KafkaActivationSpec.java

```diff
@@ -221,6 +222,14 @@ public void setTopics(String topics) {
         this.topics = topics;
     }
 
+    public String getRegexTopics() {
```

Comment:
As in the documentation the use of RegExTopics and Topics it would be good to add a check in validate and perhaps a WARNING log message if both are set to say that regextopics and topics are both set and regextopics will take precedent. That way if someone has both set they will at least get a hint as to why topics is not having any effect. Alternatively we could be brutal and throw an InvalidPropertyException stating both can not be set?

Reply:
As it is, the underlying KafkaConsumer actually takes the brutal route and throws an IllegalStateException with the message that "Subscription to topics, partitions and pattern are mutually exclusive".

Would you prefer the solution with WARNING and precedence?

Reply:
I think the precedence is done by the code already in the way the ifs are ordered. Only missing point would be the warning part.

Reply:
Precedence is done via the ifs. I think I need to test creating using the admin console or via deployment rather than annotations to see if the Kafka error is enough.